go/sdk/v4/models.go

```diff
@@ -36,7 +36,6 @@ package v4
 
 import (
   "github.com/looker-open-source/sdk-codegen/go/rtl"
-  "net/url"
```

Comment:
this is the direct cause of #765

Reply:
Ah. It doesn't load on demand like some of the other SDKs do. Good find/fix.